core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -2738,9 +2738,9 @@ public function toggle($intId=null)
 			$this->intId = $intId;
 		}
 
-		$this->strField = Input::get('field');
+		$this->strField = $strSelectorField ?: Input::get('field');
```

Comment:
Can we use `Input::post('field') ?? Input::get('field')` instead? Then we would not need the method argument.

Reply:
If `$strSelectorField` is set, we do not check for the `toggle` permission on the field (because the field is explicitly set, and selectors do not toggle). I also think "hardcoding" this is better than using "random" input that just works for the Ajax class but no other likely use case.

Reply:
that won't work, for two reasons:
 1. this is a DC action that is called via `act=toggle`, noone is passing arguments
 2. if `$strSelectorField` is set, we don't check for the toggle permission (`toggle=true` on the field), which is necessary for `act=toggle` but not for manually calling the method.